src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task {
+
+    public Deadline(String description) {
+        super(description);
+    }
+
+    @Override
+    public String getType(){
```

Comment:
```suggestion
    public String getType() {
```
This is for standardization